src/loaders/workspace_downloader/workspace_downloader_helper.py

```diff
@@ -46,18 +46,20 @@ def __init__(
         token = loader_helper.get_token(token_filepath)
         self.retrieve_sample = retrieve_sample
         self.ignore_no_sample_error = ignore_no_sample_error
+        self.ipv4 = loader_helper.get_ip()
```

Comment:
Doesn't seem like there's any need for this to be an instance variable

Reply:
👍